jcheck/src/main/java/org/openjdk/skara/jcheck/JCheckConfiguration.java

```diff
@@ -149,12 +155,13 @@ private static INI convert(INI old) {
 
     public static JCheckConfiguration parse(List<String> lines) {
         var ini = INI.parse(lines);
+        var rawConf = String.join("", lines);
```

Comment:
This isn't actually the raw conf anymore, since it's been split by lines and then put together, but the original line breaks are lost. If we are calling this the "rawConf" then we should at least put `\n` back in there. Alternatively, we could store just a hash of the lines to make it clear that it can't be used to compare to another raw source of .jcheck/conf in the future.

I think just putting linebreaks in is enough for now.

Reply:
Will add the line breaks back. Thanks!